src/panels/lovelace/common/data.ts

```diff
@@ -1,5 +1,21 @@
 import { HomeAssistant } from "../../../types";
-import { LovelaceCardConfig } from "../types";
+import { LovelaceConfig, LovelaceCardConfig } from "../types";
+
+export const migrateConfig = (hass: HomeAssistant): Promise<void> =>
```

Comment:
In a future PR, we should move this file to `src/data/lovelace.ts`, as that's also what we do for all other components.